lib/jazzy/config.rb

```diff
@@ -400,6 +401,12 @@ def print_attr_description(attr)
       end
     end
 
+    class PathnameArray < Array
```

Comment:
This simplifies `expand_paths` by providing a uniform `expand_path` method for both pathnames and arrays of pathnames. This lets us just do a simple `respond_to?(:expand_path)` check:

    def expand_paths(base_path)
      self.class.all_config_attrs.each do |attr|
        val = attr.get(self)
        if val.respond_to?(:expand_path)
          attr.set_raw(self, val.expand_path(base_path))
        end
      end
    end

The alternative is a messy little case statement in that method, which I don't think would be either easier to read or less bug-prone.